### PR TITLE
Enable all graphql queries

### PR DESCRIPTION
**Pull request by shuesken**
This makes some changes to the GraphQL setup to allow all the queries we need for the frontend to run. It also makes explicit what fields in the `Brand` table are accessible via GraphQL. Lastly, it adds CORS, making the data queryable from within the browser (allowing all origins atm).

### CODE REVIEW

File: bankgreen/settings.py [hunk 1]
```diff
@@ -142,3 +144,5 @@
 USERNAME = os.environ.get("USERNAME")
 TOKEN = os.environ.get("TOKEN")
 PASSWORD = os.environ.get("PASSWORD")
+
+CORS_ORIGIN_ALLOW_ALL = True
```
Comment (RogerTangos) [inline bankgreen/settings.py:148]: Please put this as an env import. I don't like having it in the main settings.py that will be pulled on the live site.

File: brand/models/commentary.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ class Commentary(models.Model):
     # Metadata
     brand = models.OneToOneField(
```
Comment (RogerTangos) [inline brand/models/commentary.py:20]: Should probably rename this class bank commentary since insurance brands may be in the works later on?

**Review by RogerTangos — CHANGES_REQUESTED** [review]
Cant look at everything now. Will do more later

File: bankgreen/settings.py (hunk 1 above)
Comment (shuesken) [reply bankgreen/settings.py]: done

File: brand/models/commentary.py (hunk 2 above)
Comment (shuesken) [reply brand/models/commentary.py:20]: I can do this (it'll require a bit of refactoring), but I don't think this is a useful line of thinking at this moment:
- refactoring a model name is easy
- if this app is extended include insurances ,there'll be a bunch more changes to be made anyway
- most importantly: including insurance is nowhere on any roadmap, both pre-launch and post-launch

**Comment by shuesken** [comment]
(marked this as draft for now since I'm making some more query changes to support the top3 ordering on the backend)

**Comment by shuesken** [comment]
Alright this is back to being ready for review – in setting up the connection to the frontend, I uncovered faulty parsing in `ingest_airtable` and some typos as well, so I fixed those.

**Review by RogerTangos — APPROVED** [review]
Thank you for the spelling corrections, too. I'm not able to demo this right now, but everything looks good to me at the moment.

**Pull request merged**